### PR TITLE
Replace HeaderEmbyServer with HeaderJellyfinServer on plugincatalog

### PR DESCRIPTION
**Pull request by grafixeyehero**

### CODE REVIEW

File: src/plugincatalog.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
                     <div class="selectContainer">
                         <select is="emby-select" id="selectSystem" label="${LabelDisplayPluginsFor}">
-                            <option value="Server">${HeaderEmbyServer}</option>
+                            <option value="Server">${HeaderJellyfinServer}</option>
```
Comment (JustAMan) [inline src/plugincatalog.html]: Did you check that this localizes properly? I don't see corresponding localization files' change.

Comment (cvium) [reply src/plugincatalog.html]: 'Emby' was replaced with 'Jellyfin' in all strings very early on -- even keys

Comment (dkanada) [reply src/plugincatalog.html]: Just checked, it's included in the strings.

Comment (grafixeyehero) [reply src/plugincatalog.html]: @JustAMan i checked it it's already there "HeaderJellyfinServer": "Jellyfin Server"

**Pull request merged**